ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/group_vars/all.yml

```diff
@@ -4,3 +4,17 @@ is_for_test: true
 
 avd_data_validation_mode: "error"
 avd_data_conversion_mode: "info"
+
+fabric_name: EOS_DESIGNS_DEPRECATED_VARS
+
+connected_endpoints_keys:
+  servers:
+    type: server
+  firewalls:
+    type: firewall
+  routers:
+    type: router
+  load_balancers:
+    type: load_balancer
+  storage_arrays:
+    type: storage_array
```

Comment:
I think you can move this to under host1. For host2 it can just use the defaults which are the same.

Reply:
Done